### PR TITLE
Editorial response to issue #201

### PR DESCRIPTION
**Pull request by stonematt**
fixes most of issue #201

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/219.html" title="Last updated on Aug 16, 2018, 9:59 PM GMT (a1b9f78)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/219/d7f5dd3...a1b9f78.html" title="Last updated on Aug 16, 2018, 9:59 PM GMT (a1b9f78)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2123,13 +2122,13 @@ <h3>Subject</h3>
         </p>
       </section>
       <section>
-        <h3>Signatures / Proofs</h3>
+        <h3>Proofs (aka Signatures)</h3>
```
Comment (msporny) [inline index.html]: Not quite accurate... Signatures are one type of proof. Suggest using "e.g." instead?

File: index.html [hunk 2]
```diff
@@ -2522,10 +2521,10 @@ <h3>Device Fingerprinting</h3>
 
         <p>
 There are mechanisms external to Verifiable Credentials that are used to track and
-correlate individuals on the Internet and the Web. Some of these mechanisms
-include Internet Protocol address tracking, Web Browser fingerprinting,
-Evercookies, Advertising Network trackers, mobile network position information,
-and in-application Global Positioning System APIs. The use of Verifiable
+correlate individuals on the internet and the Web. Some of these mechanisms
```
Comment (msporny) [inline index.html]: This should be capitalized.

File: index.html [hunk 3]
```diff
@@ -2522,10 +2521,10 @@ <h3>Device Fingerprinting</h3>
...
+include Internet Protocol address tracking, web browser fingerprinting,
+evercookies, advertising network trackers, mobile network position information,
+and in-application global positioning system APIs. The use of Verifiable
```
Comment (msporny) [inline index.html]: This should be capitalized.

File: index.html (hunk 1 above)
Comment (stonematt) [reply index.html]: ok, look for updates to both by EOD

File: index.html [hunk 4]
```diff
@@ -2521,10 +2521,10 @@ <h3>Device Fingerprinting</h3>
 
         <p>
 There are mechanisms external to Verifiable Credentials that are used to track and
-correlate individuals on the internet and the Web. Some of these mechanisms
-include Internet Protocol address tracking, web browser fingerprinting,
-evercookies, advertising network trackers, mobile network position information,
-and in-application global positioning system APIs. The use of Verifiable
+correlate individuals on the Internet and the Web. Some of these mechanisms
+include Internet Protocol address tracking, Web Browser fingerprinting,
```
Comment (msporny) [inline index.html]: Oops, I just intended for you to only change Internet and GPS back... Web Browser, Evercookies, and Advertising Network should have all been lower case. I'm fine w/ just pulling it in and fixing later if you don't want to do another commit.

File: index.html (hunk 1 above)
Comment (stonematt) [reply index.html]: @msporny should be done and ready to pull in.

**Comment by msporny** [comment]
LGTM. Merging.

**Pull request merged**